ha_client.py

```diff
@@ -113,7 +113,7 @@ def find_entity_attr(self, entity):
                         else:
                             unit_measur = entity_attrs['unit_of_measurement']
                     except KeyError:
-                        unit_measur = None
+                        unit_measur = ""
```

Comment:
It looks like this will break other calls to `find_entity_attr`. 
Eg on line 379-380:
```
                light_attrs = self.ha.find_entity_attr(ha_entity['id'])
                if light_attrs['unit_measure'] is None:
```

There are a few calls to this function in the Skill. I think we need a consistent approach throughout or we'll keep seeing bugs each time we change one of them.

Reply:
I missed that. I will check it again.

Reply:
Hi @krisgesling There are two approach to fix this. I just need to know what would you preferred.   'unit_measur = None'  or  'unit_measur = ""'

Reply:
Personally I think `None` is more explicit if something doesn't exist 
but if there's reasons for the other way then I'm totally open to what works best for the situation.

Reply:
With `unit_measur = ""` it's easier to fix it as we only need to change two lines. `unit_measur = None` would mean to rework some more code.

Reply:
Hi @pfefferle to keep it simple, could you also change lines 380 and 402 in `__init__.py` from `if light_attrs['unit_measure'] is None:` to `if light_attrs['unit_measure'] == ''`? It will bring less harm than keeping `None`. It will fix both issues in #65 and keep functionality to lights.

Reply:
@krisgesling @Tony763 What if we keep `None` and use something like:

```Python
        unit_measurement = self.ha.find_entity_attr(entity)
        sensor_unit = unit_measurement.get('unit_measure') or ''
```
(see line `468`)

???

But I am also fine to change everything to empty string instead of `None`.

Reply:
Hi, my option. I'm not sure if that will work correctly and also you will have to add code to check if `None` is not in other places. 

For example: 
```
        self.speak_dialog('homeassistant.set.thermostat',
                          data={
                              "dev_name": climate_attr['name'],
                              "value": temperature,
                              "unit": climate_attr['unit_measure']})
```
`climate_attr['unit_measure']` would need to have a same code to prevent Mycroft telling None as unit in dialog. (It is second part of issue #65)

Both is doable but with empty sting we will need less code to run to get a same result.

Reply:
changed!